components/Frame/Header.js

```diff
@@ -203,13 +204,17 @@ const isPositionStickySupported = () => {
 // - iOS 11.4: header is transparent and only appears after triggering a render by scrolling down enough
 const forceRefRedraw = ref => {
   if (ref) {
-    setTimeout(() => {
+    const redraw = () => {
       const display = ref.style.display
       ref.style.display = 'none'
       /* eslint-disable-next-line no-unused-expressions */
       ref.offsetHeight
```

Comment:
Off PR, but out of curiosity: What good does this do?

Reply:
It forced webkit to flush styles (render) in order to calculate `offsetHeight`.
https://stackoverflow.com/a/3485654 – I'll add a code comment